cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -193,59 +171,6 @@ CUDA_HOST_DEVICE_CALLABLE constexpr T shift(T const& val, scale_type const& scal
     return left_shift<Rep, Rad>(val, scale);
 }
 
-/** @brief Function that performs precise shift to avoid "lossiness"
- * inherent in floating point values
- *
- * Example: `auto n = fixed_point<int32_t, Radix::BASE_10>{1.001, scale_type{-3}}`
- * will construct n to have a value of 1 without the precise shift
- *
- * @tparam Rep Representation type needed for integer exponentiation
- * @tparam Rad The radix which will act as the base in the exponentiation
- * @tparam T Type for value `val` being shifted and the return type
- * @param value The value being shifted
- * @param scale The amount to shift the value by
- * @return Shifted value of type T
- */
-template <typename Rep,
-          Radix Rad,
-          typename T,
-          typename simt::std::enable_if_t<simt::std::is_integral<T>::value>* = nullptr>
-CUDA_HOST_DEVICE_CALLABLE auto shift_with_precise_round(T const& value, scale_type const& scale)
```

Comment:
Is it useful to *have* this functionality even if it is not called from the `fixed_point` constructor?   I'm not sure how it would be exposed, and it adds to testing burden, but I wanted to ask the question.

Reply:
It might be in the future - but my suggestion would be to remove it and _if_ we do need it in the future (due to a issue or feature request), we can simply got back in the git history and add the code back at that point in time.